### PR TITLE
(php_5.6.x) hotfix supported stable branch

### PR DESCRIPTION
**Pull request by AdmiringWorm**

### CODE REVIEW

File: automatic/php_5.6.x/update.ps1 [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  $lines = @(
+    @('threadsafe'; $Latest.URLTS32; $Latest.URLTS64; $Latest.ChecksumTS32; $Latest.ChecksumTS64) -join '|'
+    @('not-threadsafe'; $Latest.URLNTS32; $Latest.URLNTS64; $Latest.ChecksumNTS32; $Latest.ChecksumNTS64) -join '|'
+  )
+
+  [System.IO.File]::WriteAllLines("$PSScriptRoot\tools\downloadInfo.csv", $lines);
+}
+
+function global:au_SearchReplace {
+   @{
```
Comment (pascalberger) [inline automatic/php_5.6.x/update.ps1]: Indentation of 2 chars

File: automatic/php_5.6.x/update.ps1 [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        "$($Latest.PackageName)_5.6.x.nuspec" = @{
+            "(\<releaseNotes\>).*?(\</releaseNotes\>)" = "`${1}$($Latest.ReleaseNotes)`$2"
+        }
+    }
+}
+
+function global:au_GetLatest {
+    $download_page = Invoke-WebRequest -Uri $releases
+
+    $re      = 'php-5\.6.+-nts.+\.zip$'
```
Comment (pascalberger) [inline automatic/php_5.6.x/update.ps1]: Either align all `=` or remove leading spaces

File: automatic/php_5.6.x/tools/chocolateyUninstall.ps1 [hunk 3]
```diff
@@ -0,0 +1,20 @@
+﻿$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
```
Comment (pascalberger) [inline automatic/php_5.6.x/tools/chocolateyUninstall.ps1]: `$ErrorActionPreference = 'Stop'` ?

**Comment by AdmiringWorm** [comment]
@pascalberger I believe I fixed all requested changes

**Pull request merged**